spec/unit/resources/power_device/refresh_spec.rb

```diff
@@ -0,0 +1,46 @@
+require_relative './../../../spec_helper'
+
+describe 'oneview_test::power_device_refresh' do
+  let(:resource_name) { 'power_device' }
+  include_context 'chef context'
+
+  let(:target_class) { OneviewSDK::API200::PowerDevice }
+
+  context 'when Oneview resource can be retrieved' do
```

Comment:
I guess this could be added to a shared example. Many resources use this kind of refresh.

Reply:
Same reason that previous comment.